yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -13,13 +13,22 @@
 {% block main_content %}
 
 <div class=box>
+    <div class=edit_button>
+        {% if edit == 1 %}
```

Comment:
I'm not super sure how necessary it is to have edit = 0 or 1, it would be simpler to check in the view if the current user owns the workflow that was selected, and just pass in a boolean. It would also be a bit more readable if it was just if edit, instead of 1 or 0.